metricbeat/helper/server/http/http_test.go

```diff
@@ -88,6 +95,45 @@ func TestHTTPServers(t *testing.T) {
 	}
 }
 
+func getHandleFunc(events chan common.MapStr) func(writer http.ResponseWriter, req *http.Request) {
```

Comment:
As this is a handler function for testing maybe we can simplify it, and we could use `t.Log` for logging instead of `logp`.

Reply:
will not be an issue with `NewHttpServerWithHandler`